optuna/storages/_rdb/storage.py

```diff
@@ -64,6 +65,46 @@
 _logger = optuna.logging.get_logger(__name__)
 
 
+def _float_without_nan_to_pair_repr(value: float) -> Tuple[float, models.FloatTypeEnum]:
```

Comment:
I couldn't understand the meaning of word "pair_repr" from here. How about renaming it like "_float_without_nan_to_float_to_be_stored"?

Reply:
What about "_float_without_nan_to_stored_repr"? Does that make sense?